azurelinuxagent/common/event.py

```diff
@@ -74,6 +74,7 @@ class WALAEventOperation:
     Unknown = "Unknown"
     Upgrade = "Upgrade"
     Update = "Update"
+    HandleRemoteAccess = "RemoteAccessHandling"
```

Comment:
> HandleRemoteAccess = "Remot [](start = 3, length = 28)

Please follow the existing convention.
 1. Alphabetize
 2. Constant has the same value as the name.

Reply:
Fixed. 

---
In reply to: [194469066](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r194469066) [](ancestors = 194469066)